packages/clerk-js/src/ui/ImpersonationFab/ImpersonationFab.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+import { useCoreClerk, useCoreSession } from '../contexts';
+import { Col, Flex, Icon, Link, Text, useAppearance } from '../customizables';
+import { Portal } from '../elements/Portal';
+import { Eye } from '../icons';
+
+type EyeCircleProps = {
+  width: string;
+  height: string;
+  padding: string;
+};
+
+const EyeCircle: React.FC<EyeCircleProps> = ({ width, height, padding }) => {
+  return (
+    <Flex
+      id='impersonationEye'
```

Comment:
❓ Is this ID necessary for the appearance object? If not do we really need it?

Reply:
Need it for the eye animation.